apps/web/src/app/_components/companies/company-reviews.tsx

```diff
@@ -0,0 +1,76 @@
+"use client";
+
+import type { CompanyType } from "@cooper/db/schema";
+import { Card, CardContent, CardHeader, CardTitle } from "@cooper/ui/card";
+
+import { ReviewCardStars } from "../reviews/review-card-stars";
+
+interface CompanyReviewProps {
+  className?: string;
+  companyObj: CompanyType | undefined;
+}
+
+export function CompanyReview({ companyObj }: CompanyReviewProps) {
+  const ratings = [
+    { stars: 5, percentage: 75 },
+    { stars: 4, percentage: 15 },
+    { stars: 3, percentage: 6 },
+    { stars: 2, percentage: 3 },
+    { stars: 1, percentage: 1 },
+  ];
+
+  return (
+    <Card className="w-full max-w-lg rounded-xl border-gray-300 outline outline-[1px] outline-[#474747]">
+      <CardHeader className="border-b border-[#474747] bg-[#F7F7F7] pb-3">
+        <CardTitle className="text-base font-medium text-gray-800">
+          Reviews
+        </CardTitle>
+      </CardHeader>
+      <CardContent className="pt-6">
+        <div className="mb-6 flex items-start">
+          <div className="mr-6">
+            <h2 className="mb-4 mt-2 text-5xl">
+              5.0
+              {/* {Number(companyObj?.averageOverallRating ?? 0).toFixed(1)} */}
+            </h2>
+            <div className="my-1 flex">
+              <ReviewCardStars
+                numStars={4}
+                // numStars={Number(companyObj?.averageOverallRating)}
+              />
+            </div>
+            <p className="text-xs text-gray-500">
+              Based on {companyObj?.totalReviews ?? 0} reviews
+            </p>
+          </div>
+
+          <div className="flex-1 space-y-2 pt-1">
+            {ratings.map((rating) => (
+              <div key={rating.stars} className="flex items-center">
```

Comment:
Maybe you can make it so that when it isn't wide enough, it puts the bar graph below the star rating? I know there isn't a design for this but figured might as well.

<img width="668" alt="Screenshot 2025-03-13 at 12 51 41 PM" src="https://github.com/user-attachments/assets/eb79147c-4d6f-4ede-b9c6-24b7bb2899f0" />

If you don't want to do this no worries we can make separate dedicated mobile responsive tickets soon.